fictadvisor-web/src/components/pages/admin/admin-comments/pages/edit-comments-page/CommentsAdminEditPage.styles.ts

```diff
@@ -0,0 +1,48 @@
+import { SxProps, Theme } from '@mui/material/styles';
+
+export const header: SxProps<Theme> = {
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  padding: '16px',
+};
+
+export const title: SxProps<Theme> = {
+  borderBottom: '1px solid',
+  borderColor: 'backgroundDark.400',
+  padding: '16px',
+  width: '60%',
+  '& .MuiCardHeader-subheader': {
+    typography: 'body1',
+    color: 'grey.500',
+    display: '-webkit-box',
```

Comment:
we don't need to use vendor prefix

Reply:
it does not display answers correctly without it